chihiro_todo/todo.html

```diff
@@ -1,38 +1,62 @@
 <!DOCTYPE html>
 <html lang="ja">
-<head>
-    <meta charset="UTF-8">
-    <title>Chihiro-TODO-List</title>
-</head>
-<body>
-<h1>Chihiro-TODO-List</h1>
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="recet.css">
+        <link rel="stylesheet" href="main.css">
+        <title>Chihiro-TODO-List</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header">
+                <h1>Chihiro-TODO-List</h1>
+            </header>
+            <main class ="main_wrapper">
+                <section class="todo_wrapper">
+                    <h2>タスク管理</h2>
+                    <form class="task_form">
+                        <div>
+                            <label for="task_value" class="label">新規タスク</label>
+                            <input id="task_value" class="input" type="text" placeholder="タスクを入力して下さい" />
+                        </div>
+                        <div>
+                            <label for="person_select" class="label">担当者</label>
+                            <select id="person_select" class="select"></select>
+                        </div>
+                        <div>
+                            <button id="task_submit" class="button add_button" type="button">追加</button>
+                        </div>
+                    </form>      
+                    <div class="todo_content">
+                        <h2>Todo Tasks</h2>
+                        <ul id="todo_area" class="task_list"></ul>
+                    </div>
+                    <div class="todo_content">
+                        <h2>Doing Tasks</h2>
+                        <ul id="doing_area" class="task_list"></ul>
+                    </div>
+                    <div class="todo_content">
+                        <h2>Done Tasks</h2>
+                        <ul id="done_area" class="task_list"></ul>
+                    </div>
 
-<form>
-    <input id="task_value" type="text" />
-    <select id="person_select">
-    </select>
-    <button id="task_submit" type="button">Add Task</button>
-</form>
-<form>
-    <input id="person_name" type="text" />
-    <button id="person_submit" type="button">Add Person</button>
-</form>
-<div>
-    <h2>Todo Tasks</h2>
-    <ul id="todo_area"></ul>
-</div>
-<div>
-    <h2>Doing Tasks</h2>
-    <ul id="doing_area"></ul>
-</div>
-<div>
-    <h2>Done Tasks</h2>
-    <ul id="done_area"></ul>
-</div>
-<div>
-    <h2>Person List</h2>
-    <ul id="person_list_area"></ul>
-</div>
-<script src="main.js"></script>
-</body>
+                </section>
+                <div class="person_wrapper">
+                    <h2>担当管理者</h2>
```

Comment:
@chihiro-kakizaki 
スペルミスみたいなのありました！
```suggestion
                    <h2>担当者管理</h2>
```

Reply:
@Shibata0806 
スペルミスについても以下のコミットにて修正致しました。
[h2の担当管理者を担当者管理に修正](https://github.com/chihiro-kakizaki/todolist/pull/4/commits/0b71e93374442b32184d4849c7f1e49638a65f13)